platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -510,6 +547,26 @@ namespace platoon_strategic_ihp
             */
             bool isVehicleRightBehind(double downtrack);
             
+            /**
```

Comment:
These two new methods are described as being used for front cut-in only.  But their names and params don't indicate anything that would be restricted to a front join.  It would be nice if they be generalized enough to use for any cut-in.

Reply:
Bad description, they are used for all cut-in cases. Updated comments accordingly.